extensions/mssql/package.nls.json

```diff
@@ -174,7 +174,7 @@
 	"title.newTable": "New Table",
 	"title.designTable": "Design",
 	"title.changeNotebookConnection": "Change SQL Notebook Connection",
-	"mssql.connection.clearPooledConnections": "SQL Server: Clear Pooled Connections",
+	"mssql.connection.clearPooledConnections": "Clear Pooled Connections",
 	"mssql.parallelMessageProcessing": "[Experimental] Whether the requests to the SQL Tools Service should be handled in parallel. This is introduced to discover the issues there might be when handling all requests in parallel. The default value is false. Azure Data Studio is required to be relaunched when the value is changed.",
 	"mssql.enableSqlAuthenticationProvider": "Enables use of the Sql Authentication Provider for 'Active Directory Interactive' authentication mode when user selects 'AzureMFA' authentication. This enables Server-side resource endpoint integration when fetching access tokens. This option is only supported for 'MSAL' Azure Authentication Library. Azure Data Studio is required to be relaunched when the value is changed.",
 	"mssql.enableConnectionPooling": "Enables connection pooling on MSSQL connections to improve overall performance of Azure Data Studio connectivity. This setting is enabled by default. Azure Data Studio is required to be relaunched when the value is changed. To clear pooled connections, run the command: 'SQL Server: Clear Pooled Connections'",
```

Comment:
The old verbage is also used here, can you double check to make sure there are no other references to the "SQL Server: Clear Pooled Connections" usage?